exercises/list_links.md

```diff
@@ -24,8 +25,8 @@ Now we can **dynamically** add your favourite links to the html list.
 ### Hints
 
 - You will need to create an <code>&lt;a&gt;</code> inside of each <code>&lt;li&gt;</code></li>
-- Remember that to set the target of an `<a>` elememt you can use the `href` attribute. Without this attribute, your link will not _look_ like a link
-- The fewer code blocks you add before clicking `run` the easier it is to work out what your code will do
+- Remember that to set the target of an `<a>` elememt you can use the `href` attribute. Without this attribute, your link won't be styled like a link.
```

Comment:
it's not just that it won't be styled, but it simply won't *be* a link. Though I'm happy to consider this good enough for now

Reply:
I think can clarify the wording further here